Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+lxml = "*"
+pyusb = "*"
+pillow = "*"
+pyclipper = "*"
+pyinstaller = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.11"
```

Comment:
Is it a conflict to have the `.python-version` file set to 3.9.1 and this to 3.11?

Reply:
Good point, updated